library/src/main/res/layout/view_map.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can this be a `<merge>` tag to reduce layout depth?

Right now `MapView` extends `FrameLayout` so we have a `RelativeLayout` inside a `FrameLayout`. Perhaps `MapView` should instead extend `RelativeLayout` and then we use a `<merge>` tag here to wrap the map and find me button?